content/03.text-mining.md

```diff
@@ -7,8 +7,51 @@
+Studies have shown that these databases contain relatively precise data, but in low quantities [@doi:10.1038/nmeth1209-860; @doi:10.1038/nmeth.1284; @doi:10.1093/database/bau058; @doi:10.1093/nar/gku1205; @doi:10.1186/s12859-018-2103-8; @doi:10.1093/database/bax043].
+This happens because the high publication rate is too much for curators to keep up [@doi:10.1007/s11192-010-0202-z].
+Plus, this process isn't completely error free as databases were found to be missing annotations and entries [@doi:10.1093/bioinformatics/btm229].
+These combined findings highlight a significant need for approaches that  can scale up to an increasing publication rate and improve overall recall of relevant sentences.
+
+There have been attempts to speed up the manual curation process via semi-automatic ways [@doi:10.1016/j.jbi.2010.11.001; @doi:10.1093/database/bau067; @doi:10.1093/database/bas010; @doi:10.1145/3269206.3269229; @doi:10.1093/database/baz068; @doi:10.1186/s12859-018-2103-8; @doi:10.1186/s12859-018-2021-9].
+First step in this context is to use an automatic system to initally extract sentences from text.
+This process filters out irrelevant sentences, which means less text for  curators to sift through.
+After the pre-filtering step curators can approve or remove the identified sentences.
+This semi-automatic process was found to speed up the curation process compared to manual approach [@doi:10.1016/j.jbi.2010.11.001; @doi:10.1136/amiajnl-2013-001837].
+Despite the speed up, this process is prone to erroneous assumptions and produce bias results.
+Automated systems identify large quantities of sentences for commonly occurring relationships, but may miss out on less occurring ones [@doi:10.1016/j.jbi.2010.11.001].
+Plus, these systems could identify ambiguous sentences that even expert curators have a hard time descerning [@doi:10.1016/j.jbi.2010.11.001].
+Given these caveats, a future direction would be creating approaches that can mitigate the relationship bias.
+Futhermore, another possible direction is to look into simplifying sentence to resolve ambiguity issues.
+
+Despite the blantant negatives of manual curation, it is still needed in relationship extraction approaches.
+This process can be used to generate gold standard datasets that automated systems use for validation [@doi:10.1016/j.jbi.2012.04.004; @doi:10.1016/j.artmed.2004.07.016].
+Furthermore, manual curation can be used during the training process of  automated systems (i.e. active learning) [@doi:10.1007/s11390-012-1306-0].
+It is important to remember that manual curation alone is precise, but results in low recall rates [@doi:10.1093/bioinformatics/btm229].
+Future databases should consider initially relying on automated methods to obtain sentences at an acceptable recall level, then incorporate manual curation as a medium to fix or remove irrelevant results.
```

Comment:
Is this a traditional use of `medium` in the domain of database curation? I usually think of medium as a mode of communication and not a way to do something.